docs/ProgrammaticApi.md

```diff
@@ -0,0 +1,162 @@
+---
+id: programmatic-api
+title: Programmatic API
+---
+
+:::note
+
+The programmatic API is useful for advanced use cases only. You normally don't need to use it if you just want to run your tests.
+
+:::
+
+This page documents Jest's programmable API that can be used to run jest from `node`. TypeScript types are provided.
+
+## Simple example
+
+```js
+import jest from 'jest';
```

Comment:
this should be `import * as jest from 'jest';`, no?

Reply:
I think it depends on [`esModuleInterop`](https://www.typescriptlang.org/tsconfig#esModuleInterop) (which is recommended). I think `import * as jest from 'jest'` will always work, so changed it.